lib/chart_helpers.dart

```diff
@@ -1,8 +1,42 @@
 import 'dart:js';
 import 'package:dashboard/model.dart' as model;
 import 'package:chartjs/chartjs.dart';
+import 'package:intl/intl.dart';
```

Comment:
This doesn't seem to be used anywhere?

Reply:
Oh I'm using it for formatting the date in time series chart. Would it make sense to do library prefix (import 'package' as intl) so we know where the functions come from?

https://github.com/larksystems/katikati-reporting/pull/32/files/2b32afd982877a941cdc52a0c6c73a90e4774892#diff-1933c405783b5af5e12bd990d0e6bac3R194